openapi/data_repository_service.swagger.yaml

```diff
@@ -388,7 +414,7 @@ definitions:
           that may be used to obtain the object.
           These URIs may be external to this DRS instance.
         example:
-          drs://drs.example.org/314159
+          drs://drs.example.org/dg.1234/314159
```

Comment:
I think this is an invalid URI, since it contains a server **and** a prefix, but the rules above say it has to contain one or the other, not both.